vyxal/elements.py

```diff
@@ -3052,17 +3054,17 @@ def roman_numeral(lhs, ctx):
             raise ValueError("Number must be between 1 and 3999")
```

Comment:
This was made by copilot under my supervision and I've realised it should actually say what the problem is